src/App.tsx

```diff
@@ -1,61 +1,119 @@
 import './App.scss';
+import { useState } from 'react';
+import { TodoList } from './components/TodoList';
+import { User } from './types/User';
+import { Todo } from './types/Todo';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+
+export function getUser(userId: number): User | null {
+  const foundUser = usersFromServer.find(user => user.id === userId);
+
+  return foundUser || null;
+}
 
 export const App = () => {
+  const [userID, setValue] = useState(0);
+  const [titles, setTitles] = useState('');
+  const [savedTodos, setTodos] = useState(todosFromServer);
+  const [errorTitle, setNoTitle] = useState(false);
+  const [errorValue, setNoValue] = useState(false);
```

Comment:
```suggestion
  const [userID, setUsedID] = useState(0);
  const [titles, setTitles] = useState('');
  const [savedTodos, setSavedTodos] = useState(todosFromServer);
  const [errorTitle, setErrorTitle] = useState(false);
  const [errorValue, setErrorValue] = useState(false);
```
Please name the set function according to the variable name. This will avoid confusion and make everything clear